src/components/loader/loader.stories.tsx

```diff
@@ -54,16 +54,14 @@ export const InsideButton: Story = () => {
   const buttonContent = isLoading ? <Loader isInsideButton /> : "Click me";
   const ariaContent = isLoading ? "Loading" : "Click me";
   return (
-    <span aria-live="polite">
-      <Button
-        m={2}
-        buttonType="primary"
-        aria-label={ariaContent}
-        onClick={handleButtonClick}
-      >
-        {buttonContent}
-      </Button>
-    </span>
+    <Button
+      m={2}
+      buttonType="primary"
+      aria-label={ariaContent}
```

Comment:
Question (non-blocking): Does this need an aria-label at all when isLoading is false as there's text content at that point?

```
  const ariaContent = isLoading ? "Loading" : undefined;